src/main/java/com/ecore/roles/service/RolesService.java

```diff
@@ -0,0 +1,16 @@
+package com.ecore.roles.service;
+
+import com.ecore.roles.model.Role;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface RolesService {
+
+    Role CreateRole(Role role);
+
+    Role GetRole(UUID id);
```

Comment:
It could be just called "get" or "getById", because it's inside of the role service.